showcase/src/data/data.js

```diff
@@ -0,0 +1,31 @@
+export default [
```

Comment:
should each user have their own variable and then add that variable to this list? What will be the cleanest when this list grows very large? This is a separate issue, but I'd like a commit hook that checks that the structure is valid.